airbyte-proxy/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN apt-get update -y && apt-get install -y apache2-utils && rm -rf /var/lib/apt
 # This variable can be used to update the destintion containers that Nginx proxies to.
 ENV PROXY_PASS_WEB "http://airbyte-webapp:80"
 ENV PROXY_PASS_API "http://airbyte-server:8001"
+ENV PROXY_PASS_MICRONAUT_API "http://airbyte-server:8080"
```

Comment:
This is something that will be temporary. The server will start 2 servers: the one managed by glassfish (The old one) and the one managed by micronaut. The goal is to migrate one controller at a time in order to make it easier to review and see potential issues. Once all the controller are migrated, `8001` won't be needed anymore and it will just listen to `8080`.